website/docs/docs/build/exposures.md

```diff
@@ -24,6 +24,39 @@ Exposures make it possible to define and describe a downstream use of your dbt p
 
 Exposures are defined in `.yml` files nested under an `exposures:` key.
 
+<VersionBlock firstVersion="1.4">
+
+<File name='models/<filename>.yml'>
+
+```yaml
+version: 2
+
+exposures:
+
+  - name: weekly_jaffle_metrics
```

Comment:
We added this in the "Example" section of `docs/reference/exposure-properties.md`, and I think it's a good idea to show off a new-ish capability added in v1.3:
```
label: Jaffles by the Week
```
(snippetize me, cap'n?)